README.md

```diff
@@ -21,61 +21,157 @@ e.g.
 docker push quay.io/kiegroup/kie-cloud-operator:<version>
 ```
 
-## Deploy to OpenShift 4.5+ using OLM
+## Deploy to OpenShift 4.7+ using OLM
 
-To install this operator on OpenShift 4 for end-to-end testing, make sure you have access to a quay.io account to create
+To install this operator on OpenShift 4 for end-to-end testing, make sure you have access to a quay.io (https://quay.io/) account to create
 an application repository. Follow the [authentication](https://github.com/operator-framework/operator-courier/#authentication) 
 instructions for Operator Courier to obtain an account token. 
 This token is in the form of "basic XXXXXXXXX" and both words are required for the command.
-
-If pushing to another quay repository, replace _kiegroup_ with your username or other namespace. 
+ 
 Also note that the push command does not overwrite an existing repository, 
 and it needs to be deleted before a new version can be built and uploaded. 
 Once the bundle has been uploaded, create an [Operator Source](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#linking-the-quay-application-repository-to-your-openshift-40-cluster) 
 to load your operator bundle in OpenShift.
 
-**Create your own index image**
 
-Requires [opm](https://github.com/operator-framework/operator-registry/releases) v1.15.3+ -
+Requires [opm](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest/) 
 
-```bash
-USERNAME=tchughesiv
-VERSION=$(go run getversion.go -csv)
-IMAGE=quay.io/${USERNAME}/rhpam-operator-bundle
-BUNDLE=${IMAGE}:${VERSION}
+Requires [podman](https://podman.io/)
+
+Requires [cekit](https://github.com/cekit/cekit)
+
+**Create your own Bundle**
+Remove the following line from deploy/olm-catalog/dev/<last-version>/manifest/businessautomation-operator.clusterserviceversion.yaml
```

Comment:
there is needed empty line, otherwise title and text will be on the same line
```suggestion
**Create your own Bundle**

Remove the following line from deploy/olm-catalog/dev/<last-version>/manifest/businessautomation-operator.clusterserviceversion.yaml
```

Reply:
ok